### PR TITLE
Added utils and regex

### PR DESCRIPTION
**Pull request by nishasen**
Added some util function and regex for authentication pages

### CODE REVIEW

File: vibes/src/Utils/setDuration.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        timeObj = {...timeObj, [timeList[i]]: str};
+        str = [0, 0];
+      } else {
+        str.push(timeList[i]);
+        str.shift();
+      }
+    }
+    const timeArr = Object.values(timeObj);
+    if(Number(timeArr[0])===0) return Number(timeArr[1])+":"+timeArr[2];
+    else return Number(timeArr[0])+":"+timeArr[1]+":"+timeArr[0];
```
Comment (sadathshariff) [inline vibes/src/Utils/setDuration.js:17]: Nice Work here

**Review by sadathshariff — APPROVED** [review]
Like how you are handling validations and the duration part, learned something new.
Looks Good

**Comment by nishasen** [comment]
> Like how you are handling validations and the duration part, learned something new. Looks Good

File: vibes/src/Utils/setDuration.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    let timeObj = {H: '00', M: '00', S: '00'};
+    let str = [0, 0];
+    for(let i=0;i<timeList?.length;i++){
+      if(timeList[i]==='H'|| timeList[i]==='M' || timeList[i]==='S') {
+        str = str.join('');
+        timeObj = {...timeObj, [timeList[i]]: str};
+        str = [0, 0];
+      } else {
+        str.push(timeList[i]);
+        str.shift();
```
Comment (srejitk) [inline vibes/src/Utils/setDuration.js:12]: This was interesting approach. Good work here!

**Review by srejitk — COMMENTED** [review]
Good work here, clean code, intuitive classnames.

**Review by srejitk — APPROVED** [review]
Good work here, clean code, intuitive classnames.

**Pull request merged**